src/main.rs

```diff
@@ -5,8 +5,6 @@ use std::io::{self, Write};
 use std::path::{Component, Path, Prefix, PrefixComponent};
 use std::process::{Command, Stdio};
 
-#[macro_use]
-extern crate lazy_static;
```

Comment:
It seems `lazy_static` is not required anymore, can you please also remove it from `cargo.toml`?

Reply:
Fixed!